app/src/main/java/org/schabi/newpipe/fragments/list/BaseListFragment.java

```diff
@@ -371,6 +371,7 @@ protected void showStreamDialog(final StreamInfoItem item) {
             ));
         }
         entries.add(StreamDialogEntry.open_in_browser);
+        entries.add(StreamDialogEntry.download);
```

Comment:
Have you checked if the download option can be added to other places where `StreamDialogEntry` is used?

Reply:
On it. I'll check it out.

Reply:
Not sure about what I'm supposed to do here. @Stypox please help me understand what I should check

Reply:
I just made a similar change in #7160. You add your entry in `BaseListFragment`, but there are a bunch of other locations that have similar code (adding options to the long-press menu for stream items).

Do a search for `add(StreamDialogEntry.` and you should turn them all up.

You'll also want to test that it actually works in each other place, because some of those places may have different data available. In most cases besides `BaseListFragment`, the name of the fragment is a good indicator of where to find it in the app.

Reply:
Android Studio tells me that `StreamDialogEntry` is used in these places:
![image](https://user-images.githubusercontent.com/36421898/141614279-0e6e3f6b-517a-4692-aa72-a5febcdc29a3.png)